### PR TITLE
[DOC] Some more style changes.

### PR DESCRIPTION
**Pull request by smehringer**

### CODE REVIEW

File: test/documentation/seqan3.css [hunk 1]
```diff
@@ -131,3 +151,54 @@ table.doxtable th {
...
+
+/* automatic TOC */
+div.toc {
+    max-width: 20em;
+    width: auto;
+    align-content: center;
+}
+div.toc li {
+    font-size: 10pt;
+    font-family: -apple-system,BlinkMacSystemFont,"Segoe UI",Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
```
Comment (rrahn) [inline test/documentation/seqan3.css]: this is only on macs?

File: test/documentation/seqan3.css [hunk 2]
```diff
@@ -131,3 +151,54 @@ table.doxtable th {
...
...AEWKwXiWAneGIAABBNCly8wvBVP8fVQRUl5ffiqRc2orp2RaYmcxWTpmkKIURgzFoolUoFKKVq5hw8Xst+SaYd+Hg7/tXchBd+sRFWYBaSV07h+sm/wEmFAQChqIOz3QxdV22EYgLzF6/C44+/iJkV1aMKJ60B5CqpXQRs2w5QSsnwsLl///686veuU3vRd24Pnnt+HQgh4LF+qN4AAAd91/mNjfdxhBMO7pm7DNULHoWvsBSdF3vRebHXteaKFStcVkAI0VKpVF4E9OEEGhoackqFMQbbtmHbNlr2vYdksA2Prv0udCLBo/2AqgGS44uzCbSejyPFgPnLH8P8ZU+gsLgUhBDoug5CCCiloJRCVdUR1agQApRSnXM+NgEhRCmlVBtuQoqiuExGCAHGGFKpFGzbRuuRD3H8sx040NqPlpOX8cNHZyMc4xgIJRAMXUcslsQ9VSswb/F34PNPg4COWCwGSikcx0F27UUpRSaRZguLUqozxsYlUE4pHSHtTz/9dITJOI4Dxhi6zx3BhaMfwFdQgDUNlfAWzkRQnQGzvASz5gZQWVACwyqAoihQVXVI4oQQaJoGTdOgqqqrj8zcEpRSBUD5mAQcxymllA4dITMLLV++fNQIFK6tgfXjl8E5H+qO48BxHGRrUlGUIRKZjWeIZKQ+WnRLmxA452Xj+cC04WcBKSU0TRvVeQPTZ+QcF0KMOEtnSGSbZb6nM13XwTkvGZNAMpks1nUdw33gZrKyqqqjbnKia2Y0wBibNl4e8BuGgVx5YCovuYQQ0HUdyWSycEwCiUTCp+s6GGMTPg/fyjkhBAzDQDKZLBj1brSurs7POVcJISNupKfqXjTbn3RdRyqVssrKyry5NOCNx+MVhBBmGIaVfTN3pzTbtiGl5F6v9y4AvQBiABwCwALgsW07oGmaaG9vR0dHxx1HoKqqCqqqZgq6CAABIEoA6AAoY0xnjCmO40DX9TuOAOccUkpwzg0ANL1vki52gXA4HB4YGCC2baOiouKONKFgMKhFIpH48E9MJgAfAF99ff16v9//XD4fwG93k1LyUCi0taWl5W0A0XSPZTKNJ92tNCGa5eBkKvcNwEl3G0ASQAJAPO3EUhkWkcws+9LSGppqbfC0w3IALE2CjfmVMp0ftHRXp1gDGQIO/hfbvwF6ZBhg2gmmVgAAAABJRU5ErkJggg==);
```
Comment (rrahn) [inline test/documentation/seqan3.css]: ?

File: test/documentation/seqan3.css (hunk 1 above)
Comment (smehringer) [reply test/documentation/seqan3.css]: probably.. this is the default font in github so I just copy and pasted this.

File: test/documentation/seqan3.css (hunk 2 above)
Comment (smehringer) [reply test/documentation/seqan3.css]: this adds the paper & pencil icon from seqan2 to the assignment header 😁 💅

File: test/documentation/seqan3.css (hunk 1 above)
Comment (rrahn) [reply test/documentation/seqan3.css]: ok

Comment (rrahn) [reply test/documentation/seqan3.css]: but this won't induce issues on non apple systems?

File: test/documentation/seqan3.css (hunk 2 above)
Comment (rrahn) [reply test/documentation/seqan3.css]: Ok, but why this wired hash?

File: test/documentation/seqan3.css (hunk 1 above)
Comment (smehringer) [reply test/documentation/seqan3.css]: it is already in the css for the normal page content, so does it? How does the documentation look like on your mac ? :wink:

File: test/documentation/seqan3.css (hunk 2 above)
Comment (smehringer) [reply test/documentation/seqan3.css]: How do I know :sweat_smile: I literally have no CSS experience. I just "inspect the page" in my browser and then copy and paste what I find... (When you inspect the icon in seqan2, this is hash is what I get. And it works :grin: If you tell me how I can inlcude an image in css AND how doxygen may find it with his autogenerated html pages I'll gladly change it to some proper file location.)

Comment (smehringer) [reply test/documentation/seqan3.css]: I suspect that the url will become invalid if we take down the seqan2 readTheDocs page but we are not planning to are we?

Comment (eseiler) [reply test/documentation/seqan3.css]: The string is the encoded image and no url. You can paste it to a png base 64 converter and you'll get the image.

Comment (rrahn) [reply test/documentation/seqan3.css]: Ok, it also seems to be a good practice for css in order to save http requests.

File: test/documentation/seqan3.css (hunk 1 above)
Comment (rrahn) [reply test/documentation/seqan3.css]: We'll see and fix it later if it is broken somewhere

**Pull request merged**